core/Updates/2.15.1-b1.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Updater;
+use Piwik\Updates;
+
+
+class Updates_2_15_1_b1 extends Updates
+{
+    public function getMigrationQueries(Updater $updater)
+    {
+        $updateSql = array(
+            'ALTER TABLE `' . Common::prefixTable('user_language')
```

Comment:
You mean the `LanguagesManager` plugin. Thought about putting it there first. But I hadn't found any other plugins in core having update files, so I put it to core. Should we move it?